pretext/LinearBasic/Summary.ptx

```diff
@@ -37,7 +37,7 @@
             </li>
             <li>
                 <p>The fundamental operations for a deque are <c>push_front</c>, <c>push_back</c>, <c>pop_front</c>, <c>pop_back</c>, and <c>empty</c>.</p>
-            </li
+            </li>
```

Comment:
This change was fixed in #333. Probably should sync your forked repo and pull again.